File: python/wash_hands/wash_hands.py
__pdoc__ = {'main': False}


def wash_hands(N, nM):
    sec = N*21*nM*30
    return '{} minutes and {} seconds'.format(sec//60, sec % 60)


def main():
    x = 8
    y = 7
    answer = wash_hands(x, y)
    print(answer)


if __name__ == '__main__':
    main()

File: javascript/washHands/washHands.js
const washHands = (N, nM) => {
  const properHandWashing = 21;
  const dailyHandWashing = N * properHandWashing;
  const monthlyHandWashing = dailyHandWashing * nM * 30;
  const min = Math.floor(monthlyHandWashing / 60);
  const sec = monthlyHandWashing % 60;
  const totalHandWashingTime = `${min} minutes and ${sec} seconds`;
  return totalHandWashingTime;
};

module.exports.washHands = washHands;
